release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md

```diff
@@ -40,6 +40,26 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.2.2](https://git
 
 ## Developer tools updates
 
+### New features
+
+#### Language Server
+
+- Implemented a new code action to extract a constant expression to a constant
+- Added a new code action to convert an array to array mapping into a query expression
+- Introduced new API to load a project into the Workspace Manager
```

Comment:
@gayalkuruppu we added following new features as well:
1. Extract to function
2. Extract to local variable
3. Rename popup for create variable code action <- Let's add this under a new feature